uint8 ID
float32[4] CUR
float32[4] RPM